src/main/scala/cromwell/engine/RuntimeAttribute.scala

```diff
@@ -0,0 +1,9 @@
+package cromwell.engine
+
+object RuntimeAttribute extends Enumeration {
+  type RuntimeAttribute = Value
+  val failOnStderr = Value("failOnStderr")
+  val docker = Value("docker")
+  val serialize = Value("serialize")
+  val memory = Value("memory")
+}
```

Comment:
Yay for this, but it looks like there's still a `RuntimeAttribute` in the `package.scala` that's used pretty much everywhere else in the code that duplicates what this does.